interpreter.go

```diff
@@ -50,7 +59,7 @@ func (err RuntimeError) Error() string {
 
 // Values and state
 
-type bindingFrame map[*identifier]thunk
+type bindingFrame map[identifier]*thunk
 
 type value interface {
```

Comment:
This is a pretty generic name across what is becoming a large package.  We might want to break out the various stages into their own packages.  This would make some innards exported but it might be worth it.  Something to think about later, I guess.